src/django_aetos/views.py

```diff
@@ -18,12 +19,45 @@ def export_metrics(request):
                 metric["add_help"] = True
             known_metrics.add(metric["name_without_labels"])
             metrics.append(metric)
+    
+    return metrics
+
+def check_auth(request):
+    ip_address = request.META['REMOTE_ADDR']
```

Comment:
`REMOTE_ADDR` will not be correct, if django is behind a reverse proxy (so 99% of the time). Document somewhere that an additional app like [`django-xff`](https://pypi.org/project/django-xff/) might be needed.

Reply:
Added Info in 54dd922650a9dd91e7590ea2d97fd2fe899ee9ea